airbyte-integrations/connectors/source-square/source_square/source.py

```diff
@@ -367,12 +363,18 @@ def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, Any]]]:
                 "No locations found. Orders cannot be extracted without locations. "
                 "Check https://developer.squareup.com/explorer/square/locations-api/list-locations"
             )
-            exit(1)
+            return [None]
```

Comment:
Is `[None]`  for acceptance tests ?

Reply:
maybe we should return empty stream since there is nothing to procced with

Reply:
I was looking at the stream_slices at core.py and it was returning [None] by default. Change it to `yield from []`